<launch>


    <node name="rosbag" pkg="rosbag" type="play" respawn="false" args="--delay=1 --queue=1000 /media/ubuntu/Tony-hd/bag/indoor1.bag" />

    <node pkg="visionworks" type="feature_tracker" name="feature_tracker" output="screen">
        <remap from="~image_raw" to="/bluefox2/a"/> 
        <!--remap from="~image_raw" to="/videofile/image_raw"/--> 

        <param name="harris_k" type="double" value="0.04"/> 
        <param name="harris_thresh" type="double" value="100"/> 
        <param name="harris_cell_size" type="int" value="18"/> 
        <param name="array_capacity" type="int" value="200"/> 

        <param name="pyr_levels" type="int" value="3"/> 
        <param name="lk_num_iters" type="int" value="30"/> 
        <param name="lk_win_size" type="int" value="21"/> 
        <param name="calib_file" type="string" value="$(find visionworks)/config/04_camera_calib.yaml"/> 
        <param name="ransac_thresh" type="double" value="0.5"/> 
    </node>

	

</launch>
